api/src/routes/settings.js

```diff
@@ -0,0 +1,74 @@
+const db = require('../db/connection')
+const { cache } = require('../config')
+const { validateRole } = require('../utils/roles')
+
+/**
+ * Retrieve app settings from the database
+ */
+async function get (req, res) {
+  try {
+    const { user } = req
+
+    // only admins are allowed to view settings
+    if (!user || !user.roles || !validateRole(user.roles, 'admin')) {
+      return res.boom.unauthorized('Not authorized')
+    }
+
+    // Get data from the cache first
+    let settingsMap = {}
+    let data = JSON.parse(cache.exportJson())
```

Comment:
Consider using cache.keys() and then iterate them and cache.get(...) to extract the settings instead of JSON export + parsing.  However, if this `get()` function is only called once, then it may not be worth refactoring and it would be fine to leave as is.

Reply:
This has been rewritten